lxd/storage/load.go

```diff
@@ -34,23 +34,19 @@ func volIDFuncMake(state *state.State, poolID int64) func(volType drivers.Volume
 		// format <project>_<volume>. However not all volume types currently use this
 		// encoding format, so if there is no underscore in the volume name then we assume
 		// the project is default.
-		project := "default"
+		projectName := "default"
```

Comment:
Wouldn't it make sense to use `project.Default` here as well?

Reply:
@monstermunchkin Yep! :)